internal/docker/docker.go

```diff
@@ -159,7 +166,14 @@ func (c *Docker) Build(ctx context.Context, request *BuildRequest) error {
 		fmt.Sprintf("--library-id=%s", request.LibraryID),
 	}
 
-	return c.runDocker(ctx, request.Cfg, CommandBuild, mounts, commandArgs)
+	if err := c.runDocker(ctx, request.Cfg, CommandBuild, mounts, commandArgs); err != nil {
+		return err
+	}
+
+	// It's OK to not handle this error since this file is ephemeral.
+	os.Remove(jsonFilePath)
```

Comment:
Maybe use Go's [`defer` statement](https://go.dev/tour/flowcontrol/12) to schedule the cleanup? It is idiomatic in Go and has the benefit of guaranteed clean up on all error path, including `c.runDocker()`  in L170.
Sample usage in this repo [here](https://github.com/googleapis/librarian/blob/ade9ce0bb270281213252159fb81576bd29ca96f/internal/github/github.go#L108)

Reply:
Done.